electronic_store/data/ir_model_fields.xml

```diff
@@ -0,0 +1,64 @@
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="ttype">datetime</field>
+    </record>
+    <record id="field_service_engineeres" model="ir.model.fields">
+        <field name="name">x_service_engineers</field>
+        <field name="field_description">Service Engineers</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="ttype">char</field>
+        <field name="readonly" eval="True"/>
+    </record>
+    <record id="x_comments_remarks_record" model="ir.model.fields">
+        <field name="name">x_comments</field>
+        <field name="field_description">Remarks</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="ttype">html</field>
+    </record>
+    <record id="x_project_task_record" model="ir.model.fields">
+        <field name="name">x_project_task_id</field>
+        <field name="field_description">Task</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="on_delete">cascade</field>
+        <field name="relation">project.task</field>
+        <field name="required" eval="True"/>
+        <field name="ttype">many2one</field>
+    </record>
+    <record id="x_name_record" model="ir.model.fields">
+        <field name="name">x_name</field>
+        <field name="field_description">Name</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="related">x_project_task_id.name</field>
+        <field name="ttype">char</field>
+    </record>
+    <record id="field_products_id_worksheet_template" model="ir.model.fields">
+        <field name="name">x_products</field>
+        <field name="field_description">Products on Tasks</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="ttype">boolean</field>
+        <field name="related">x_project_task_id.allow_material</field>
+    </record>
+    <record id="field_type_of_installation" model="ir.model.fields">
+        <field name="name">x_type_of_installation</field>
+        <field name="field_description">Type of Installation</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="ttype">selection</field>
+        <field name="selection">[('air_conditioner', 'Air Conditioner'), ('refrigerator', 'Refrigerator')]</field>
+    </record>
+    <record id="field_makemodel" model="ir.model.fields">
+        <field name="name">x_makemodel</field>
+        <field name="field_description">Model Number</field>
+        <field name="model_id" ref="x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="ttype">char</field>
+    </record>
+    <record id="new_field_to_set_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_days</field>
```

Comment:
Looking at the values, it would make way more sense to have months instead of days. Could you adapt the code and the values?
```suggestion
        <field name="name">x_warranty_months</field>
```